addons/policies/bootstrap_entity_policies.json

```diff
@@ -2120,7 +2120,8 @@
                 [
                     "admin",
                     "service-account-atlan-argo",
-                    "service-account-atlan-backend"
+                    "service-account-atlan-backend",
+                    "atlan-governance-workflows"
```

Comment:
This will allow atlan-governance-workflows user to UPDATE ANY custom metadata on ANY asset in Atlan, just making sure you are aware of it

Reply:
Yes, this is one the intended behaviours.